### PR TITLE
added graham2021levit 2

### PR DESCRIPTION
**Pull request by TongruiLi**

### CODE REVIEW

File: _summaries/graham2021levit_2.md [hunk 1]
```diff
@@ -1,18 +1,56 @@
...
+
+The author claimed to acheive relatively good result (80% or more on ImageNet classification) while remaining extremely fast.
+
+In comparsion to other efficient models, LeVit net greatly improve efficiency while only having 25% to 75% more parameter than previous models.
+
 * What interesting variants are explored?
 
+![ablation](graham2021levit_2a.png)
+
+The autho also performed ablation study with different components removed. Without pyramid shape the number of parameters takes a drastic hit along the accuracy. Other variations shows similar performance and paramter size with the base model.
```
Comment (zhaoyue-zephyrus) [inline _summaries/graham2021levit_2.md:52]: typo: autho -> author

File: _summaries/graham2021levit_2.md [hunk 2]
```diff
@@ -1,18 +1,56 @@
...
+
 ## TL;DR
```
Comment (zhaoyue-zephyrus) [inline _summaries/graham2021levit_2.md:54]: missing TL;DR?

File: _summaries/graham2021levit_2.md [hunk 3]
```diff
@@ -1,18 +1,56 @@
...
+
+1. *Patch Embedding* small convnet is applied on input before transofmrer
+2. *No classification token* replace classification token with averagepooling to use BCHW token
+3. *Normalization* use ViT's normalization technique plus batch norm after every convolutin layer, which provides **runtime advantage in inference time**
+4. *Multi resolution pyramid* Decrese conv layer's activation map while increasing channel size. 
+5. *Downsampling* shrinkibng attention block before Q activation
+6. *Attention bias* replaces positional embedding, which allows injection of position information in attention
+7. *Smaller Keys* Attention bias reduce pressure on location information, which means that we can safely reduce V matrix's size.
+8. *Attention Activation* use Hardwish activation to $$A^hV$$, which is similar to ResNet bottleneck
+8. *Reducing MLP blocks* reduce expansion factor of convolution to minimize compution cost
```
Comment (zhaoyue-zephyrus) [inline _summaries/graham2021levit_2.md:37]: 8 -> 9

**Review by zhaoyue-zephyrus — COMMENTED** [review]
Nice draft! I especially like the concise summary of main changes. There are a few typos and you may complete the TL;DR when you have time.
